docs/reference-guides/data/data-core.md

```diff
@@ -150,7 +150,7 @@ _Parameters_
 
 _Returns_
 
--   `undefined< 'edit' >`: Current user object.
+-   `ET.User< 'edit' >`: Current user object.
```

Comment:
Nit: Maybe as a follow-up, we could add a clarification of what `ET` is. It's nowhere to be found in the doc file.

Reply:
Yes, the types are currently just names without meaning or explanation. Solving it is a multi-step task. First we need to generate docs also for the `@template` type arguments, docs for types, somehow cross-reference them etc.